homeassistant/components/qnap_qsw/sensor.py

```diff
@@ -55,6 +70,40 @@ class QswSensorEntityDescription(SensorEntityDescription, QswEntityDescription):
         state_class=SensorStateClass.MEASUREMENT,
         subkey=QSD_FAN2_SPEED,
     ),
+    QswSensorEntityDescription(
```

Comment:
I expect these will put quite a bit of data in the database.  Should these be disabled by default?

Reply:
@bdraco yeah, probably, I can disable all of these by default :)

(Excuse me for not replying before, but I'm currently on a trip)